<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::Application::MainWindow</class>
 <widget class="QMainWindow" name="Elpida::Application::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elpida</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSystemInfo">
       <attribute name="title">
        <string>System information</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="gbSystemInfo">
          <property name="title">
           <string>System information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,1">
           <item row="2" column="0">
            <widget class="QGroupBox" name="gbOverheadsInfo">
             <property name="title">
              <string>Overheads information</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="gbCpuInfo">
             <property name="title">
              <string>CPU information</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="gbOsInfo">
             <property name="title">
              <string>Os information</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="gbMemoryInfo">
             <property name="title">
              <string>Memory information</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2" rowspan="4" colspan="2">
            <widget class="QGroupBox" name="gbTopology">
             <property name="title">
              <string>Topology</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QWidget" name="wTopologyContainer" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabBenchmarks">
       <attribute name="title">
        <string>Benchmarks</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QWidget" name="wBenchmarks" native="true">
          <layout class="QGridLayout" name="gridLayout_2"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_results_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_results_as">
   <property name="text">
    <string>&amp;Save results as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
